README.md

```diff
@@ -1,32 +1,69 @@
-# Used  - An AI powered app, that automtically generates posts on selling platforms. All you need to take is a picture.
+# Java AI - AI Shop
 
-Run the sample locally or within GitHub CodeSpaces.
```

Comment:
Please add back the CodeSpaces run, to enable developers to try it on the fly also with flacky internet connection